src/config/database.js

```diff
@@ -0,0 +1,16 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import Sequelize from "sequelize";
+
+export const sequelize = new Sequelize(process.env.DEV_DATABASE_URL, {
```

Comment:
you have config file then again you are calling env, use config file to determine node environment